src/routes/landing/components/Heading.js

```diff
@@ -1,20 +1,110 @@
 // @flow
 
 import * as React from 'react'
-
+import { TouchableOpacity } from 'react-native'
+import Toast from 'react-native-simple-toast'
 import LocationBig from '../assets/LocationBig.png'
 import styled, { type StyledComponent } from 'styled-components/native'
+import AppSettings from '../../../modules/settings/AppSettings'
+import { baseUrl, liveBaseUrl, testBaseUrl } from '../../../modules/endpoint/constants'
+import { Button } from 'react-native-elements'
 import type { ThemeType } from '../../../modules/theme/constants/theme'
+import moment from 'moment'
+import type Moment from 'moment'
+
+const API_URL_OVERRIDE_MIN_CLICKS = 10
+const CLICK_TIMEOUT = 8
+
+type StateType = {| clickCount: number, clickStart: ?Moment, apiUrlOverride: ?string |}
+
+type PropsType = {|
+  clearResourcesAndCache: () => void,
+  theme: ThemeType
+|}
+
+const ApiUrlText = styled.Text`
+padding-top: 10px;
+color: red;
+`
 
 const LocationImage: StyledComponent<{}, ThemeType, *> = styled.Image`
   height: 70px;
   resize-mode: contain;
 `
 
-class Heading extends React.Component<{||}> {
+const Wrapper: StyledComponent<{| children: React.Node |}, {}, *> = styled.View`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+class Heading extends React.Component<PropsType, StateType> {
+  constructor (props: PropsType) {
+    super(props)
+    this.state = { clickCount: 0, apiUrlOverride: null, clickStart: null }
+  }
+
+  componentDidMount () {
+    const appSettings = new AppSettings()
+    appSettings.loadApiUrlOverride().catch(() => {}).then(apiUrlOverride => this.setState({ apiUrlOverride }))
+  }
+
+  onImagePress = async () => {
```

Comment:
I think it would be cool if this would be extracted into a separate Component. For example an `EastereggImage`

Reply:
I'm not sure whether this really is practical. `Heading` would then almost be empty and have no functionality.

Reply:
Done